apps/web/ee/lib/workflows/reminders/emailReminderManager.ts

```diff
@@ -0,0 +1,146 @@
+import {
+  TimeUnit,
+  WorkflowTriggerEvents,
+  WorkflowTemplates,
+  WorkflowActions,
+  WorkflowMethods,
+} from "@prisma/client";
+import client from "@sendgrid/client";
+import sgMail from "@sendgrid/mail";
+import dayjs from "dayjs";
```

Comment:
```suggestion
import dayjs from "@calcom/dayjs";
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix